flask/run.sh

```diff
@@ -5,18 +5,15 @@ set -e
 # This is not a standalone script. It is called from ../deploy.sh that
 # sets up the right environemnt variables and files for it.
 
-if [ ! -d ./venv ]; then 
+if [ ! -d ./venv ]; then
     python3 -m venv ./venv
 fi
 source venv/bin/activate
 pip3 install -r requirements.txt
 
 function cleanup {
-  stop.sh python3 $LOCAL_PORT 
-  if [[ "$VIRTUAL_ENV" != "" ]]; then
-    deactivate
-  fi
+  stop.sh python3 $LOCAL_PORT
 }
 trap cleanup EXIT
 
-PORT=$LOCAL_PORT python3 run.py
+flask run --port $LOCAL_PORT
```

Comment:
should we then also get rid of `run.py` and [this section](https://github.com/sentry-demos/empower/blob/master/flask/src/main.py#L238C11-L244) now that they are no longer used?

Reply:
ah I did delete run.py but I'll delete that too